packages/web-react/src/components/Dropdown/stories/DropdownPlacements.tsx

```diff
@@ -0,0 +1,37 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
+import { Stack } from '../../Stack';
+import { dropdownContent, dropdownTrigger } from './Dropdown';
+import { IconsProvider } from '../../../context';
+import { SpiritDropdownProps } from '../../../types';
+import Dropdown from '../Dropdown';
```

Comment:
Do we have a sorting convention for imports? Typically they are ordered from the farthest to the closest.

Reply:
No, but I think that there is an existing rule for this. I will look for it.